CHANGELOG.md

```diff
@@ -25,15 +32,6 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - package version
 
-## [3.0.5] - 2020-09-03
-
-### Added
-
-- custom methods
-- templateId
-- templateAlias
-- variables
-
```

Comment:
What this removed by mistake? 🤔

Reply:
No, the 3.0.5 is doubled in the changelog, this one is wrong.